services/py-api/src/service/participants_service.py

```diff
@@ -43,15 +43,11 @@ async def _create_random_participant_in_transaction(
         self, input_data: ParticipantRequestBody, session: Optional[AsyncIOMotorClientSession] = None
     ) -> Result[Tuple[Participant, Team], DuplicateEmailError | DuplicateTeamNameError | Exception]:
 
-        team = None
-        if is_err(team):
-            return team
-
-        participant = await self._participant_repo.create(input_data, session, team_id=team.ok_value.id)
+        participant = await self._participant_repo.create(input_data, session, team_id=None)
         if is_err(participant):
             return participant
 
-        return Ok((participant.ok_value, team.ok_value))
+        return Ok((participant.ok_value, ""))
```

Comment:
I am not sure on what to return here since the random participant shouldn't have a team when he is initially registered